src/clients/python/sams_client/endpoints/assets.py

```diff
@@ -54,6 +55,32 @@ def get_binary_by_id(
             callback=callback
         )
 
+    def get_binary_zip_by_id(
+        self,
+        item_ids: [ObjectId] = None,
+        headers: Dict[str, Any] = None,
+        callback: Callable[[requests.Response], requests.Response] = None
+    ) -> requests.Response:
+        """Helper method to get asset binaries by its ids
+
+        :param bson.objectid.ObjectId item_id: The ID of the asset
+        :param dict headers: Dictionary of headers to apply
+        :param callback: A callback function to manipulate the response
+        :rtype: requests.Response
+        :return: The compressed binaries zip, if found
+        """
+
+        if not self._read_binary_zip_url:
+            return self._return_405()
+        return self._client.get(
+            url='{}/{}'.format(
+                self._read_binary_zip_url,
+                str(item_ids)
```

Comment:
Converting `List[ObjectId]` would result in something like the following:
`"[ObjectId('5f61a9db066778567faf2040'), ObjectId('5f61a9db066778567faf2041')]"`

Instead you could generate a comma separated list, i.e.
`','.join(str(item_id) for item_id in item_ids)`
Which would result with the following instead:
'5f61a9db066778567faf2040,5f61a9db066778567faf2041'